index.html

```diff
@@ -391,7 +391,7 @@ <h2 class="section-title">Get in Touch</h2>
         <div class="container">
             <h2 class="section-title">Source Code</h2>
             <div class="section-content" style="text-align: center;">
-                <p style="font-size: 1.2rem; margin-bottom: 30px;">Unlike TrueNAS CORE, zVault is 100% Open Source and is not encumbered with any additional proprietary licenses like the TrueNAS Enterprise License Agreement. Check out our repositories and contribute to the project.</p>
+                <p style="font-size: 1.2rem; margin-bottom: 30px;">Unlike TrueNAS CORE, zVault is 100% open source and is not encumbered with any additional proprietary licenses like the TrueNAS Enterprise License Agreement. Check out our repositories and contribute to the project.</p>
```

Comment:
```suggestion
                <p style="font-size: 1.2rem; margin-bottom: 30px;">Unlike TrueNAS Legacy (CORE), zVault is 100% open source and is not encumbered with any additional proprietary licenses like the TrueNAS Enterprise License Agreement. Check out our repositories and contribute to the project.</p>
```

CORE is now _Legacy_ in the footer, which is probably site-wide; <https://www.truenas.com/truenas-community-edition/#menu-useful-links> and elsewhere.

Reply:
Do not bring confusion here.

People now know and understand that there are two worlds in iXsystems universe.

The **TrueNAS SCALE** - Linux based.

The **TrueNAS CORE** - FreeBSD based.

Lets leave all the nuances out of this game - **Legacy**, **Community Edition**, they can come up with any new (re)names they want - but its either **TrueNAS SCALE** or **TrueNAS CORE**.